### PR TITLE
improvements to the backup scripts

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
This is ready :)

**Comment by szaimen** [comment]
Now its ready :)

**Comment by enoch85** [comment]
Is it ready now? :)

**Comment by szaimen** [comment]
Yes :)

File: not-supported/backup-viewer.sh [hunk 1]
```diff
@@ -150,6 +150,14 @@ then
     fi
 fi
 
+# Check if pending snapshot is existing a second time and cancel the viewing in this case.
+if does_snapshot_exist "NcVM-snapshot-pending"
+then
+    msg_box "The snapshot pending does exist. Can currently not show the backup.
```
Comment (enoch85) [inline not-supported/backup-viewer.sh:156]: Does it exist or not? :)

Comment (szaimen) [reply not-supported/backup-viewer.sh:156]: You mean the snapshot?

The "snapshot-pending" does exist, hence you cannot show the backup. This is correct.
I guess the wording isn't that clear...

**Pull request merged**